<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dImport</class>
 <widget class="QDialog" name="dImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Import Games Wizard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color: rgb(30, 31, 34);}

QFrame{
background-color: rgb(38,39,42);
border:1px solid;
border-color:rgb(33, 34, 43);}

QLabel{
color: rgb(243, 244, 245);
border:none;}

QPushButton{
background-color: rgb(43,45,49);
color:rgb(243, 244, 245);
border:1px solid;
border-color:rgb(64,66,73);}
QPushButton:hover:!pressed
{
background-color: rgb(54, 55, 61);
}

QTextEdit{
background-color:rgb(130,133,143);
color: rgb(243, 244, 245);
border:none;}

QPlainTextEdit{
color: rgb(130,133,143);
border:none;}

QListWidget{
background-color:rgb(130,133,143);
color: rgb(243, 244, 245);
border:1px solid;
border-color:rgb(64,66,73);}

QTableWidget{
background-color:rgb(130,133,143);
color: rgb(243, 244, 245);
border:1px solid;
border-color:rgb(64,66,73);}

QToolButton{
background-color: rgb(43,45,49);
color:rgb(243, 244, 245);
border:1px solid;
border-color:rgb(64,66,73);}
QToolButton:hover:!pressed
{
background-color: rgb(54, 55, 61);
}

QCheckBox{
background-color:none;
color: rgb(243, 244, 245);
border:none;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <widget class="QPushButton" name="btnNext">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancel">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="swImportGames">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="page_1">
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>721</width>
         <height>421</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>110</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Autofull from path:</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="tbFormat">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>301</width>
          <height>25</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;{game}-{version}&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="tbExt">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>230</y>
          <width>301</width>
          <height>25</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;.exe&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Executable Extension:</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>280</y>
          <width>431</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Search for game information in the local metadata datbase (recommended)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>310</y>
          <width>431</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Download Banners</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>340</y>
          <width>431</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Download metadata from VNDB</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>140</y>
          <width>461</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Use &quot;/&quot; for subfolders and &quot;-&quot; for main folder. Use &quot;_&quot; for spaces. See examples below:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>160</y>
          <width>371</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Come_Home-v1.0_Taboo   {game}-{version}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>180</y>
          <width>371</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Come_Home/v1.0_Taboo   {game}/{version}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAdd_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>70</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Folder</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="tbPath">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>70</y>
          <width>301</width>
          <height>25</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;D:\Games&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Game Path</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>370</y>
          <width>431</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Copy to Hydrus95 Game folder</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbtest">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>310</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>741</width>
         <height>451</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTableWidget" name="twGames">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>681</width>
          <height>281</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>681</width>
          <height>0</height>
         </size>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Version</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Extension</string>
         </property>
        </column>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>631</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="plainText">
         <string>Ready to Import</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>661</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>This wizard is ready to import the games below. Please review the list to confirm that the correct games will be imported,
and make any necessary changes. Change the game names in the Name column or remove items with the Delete key.</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
